packages/flutter_tools/test/commands.shard/hermetic/clean_test.dart

```diff
@@ -35,16 +35,10 @@ void main() {
     group('general', () {
       MemoryFileSystem fs;
       Directory buildDirectory;
-      FlutterProject projectUnderTest;
 
-      setUp(() {
-        fs = MemoryFileSystem.test();
-
-        final Directory currentDirectory = fs.currentDirectory;
-        buildDirectory = currentDirectory.childDirectory('build');
-        buildDirectory.createSync(recursive: true);
-
-        projectUnderTest = FlutterProject.fromDirectory(currentDirectory);
+      FlutterProject setupProjectUnderTest(Directory currentDirectory) {
```

Comment:
`FlutterProject.fromDirectory` can't be run anymore outside of a `testUsingContext`.